src/Utils.js

```diff
@@ -161,13 +162,14 @@ export function withSetStateAllowed(fn) {
   }
 }
 
-export function splitSelector(selector) {
-  return selector.split(/(?=\.|\[.*\])|(?=#|\[#.*\])/);
+export function getAst(selector) {
```

Comment:
This name is pretty broad for just dealing with CSS selectors - maybe `getSelectorAST`?